tests/wasi/wasi_test.go

```diff
@@ -78,3 +78,61 @@ func Test_ArgsSizesGet_ArgsGet(t *testing.T) {
 		})
 	}
 }
+
+//go:embed testdata/environ.wasm
+var environWasm []byte
+
+// Test environ_sizes_get and environ_get. environ_sizes_get must be used to know the length and the
+// size of the result of environ_get, so TinyGo calls the both of them together to retrieve the
+// WASI's environment variables. Any other language runtime would do the same things.
+func Test_environ_sizes_get_environ_get(t *testing.T) {
```

Comment:
too bad we can't have a separate func per wasi here as this is really hard to read vs `Test_environ_sizes_get` and `Test_environ_get`. Any way we can do that, as if we can it is also easier to count which things are done by virtue of the test results